tests/lac/vector_move_operations.cc

```diff
@@ -0,0 +1,82 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2012 - 2015 by the deal.II authors
```

Comment:
ah, forget it -- I forgot that you make this happen in the name of the output file.